src/app/stack/stack-analyses.service.ts

```diff
@@ -45,11 +45,13 @@ export class StackAnalysesService {
       if (params['access_token']) {
         let headers: Headers = new Headers();
         headers.append('Authorization', 'Bearer ' + params['access_token']);
+        headers.append('x-3scale-account-secret','not-set');
```

Comment:
Probably this can be under a conditional param. 
```suggestion
        if (params['devcluster'])
            headers.append('x-3scale-account-secret','not-set');
```

Reply:
fixed